### DISCUSSION TITLE
Rename Model struct when generating entities?

### DISCUSSION THREAD

**Discussion by affanshahid** [SeaQL/sea-orm#1683] [Q&A]
Is it possible to rename the model struct when generating entites? I'm using `poem-openapi` and this causes a conflict since every model is called `Model`. Poem does allow renaming using attributes but I can't configure a difference attribute value through the sea-orm-cli either.

**Comment by billy1624** [comment]
Hey @affanshahid, note that each entity should be place under a separate module
https://github.com/SeaQL/sea-orm/tree/master/tests/common/bakery_chain
